crates/bevy_core_pipeline/src/oit/resolve/mod.rs

```diff
@@ -89,6 +97,26 @@ pub fn is_oit_supported(adapter: &RenderAdapter, device: &RenderDevice, warn: bo
     true
 }
 
+/// System set for systems used for OIT resolve.
+#[derive(SystemSet, PartialEq, Eq, Hash, Debug, Clone)]
+struct OitResolveSystems;
+
+/// A resource to indicate that OIT resolve is supported.
+#[derive(Resource)]
+struct OitResolveSupported;
+
+/// A system to perform a one-time check for whether OIT resolve is supported and insert a resource
+/// if so.
+fn check_is_oit_supported(
+    mut commands: Commands,
+    render_device: Res<RenderDevice>,
+    render_adapter: Res<RenderAdapter>,
+) {
+    if is_oit_supported(&render_adapter, &render_device, true) {
```

Comment:
It might require a bit more work, but this function is only used in one other place, I think we could instead inline it here and pipe the resource to that other place that needs to know if OIT is supported.

Reply:
I would like to make this a followup rather than blocking this here. Minimal diff is better for these refactors IMO!

Reply:
Especially in rendering code it's better to avoid sprawling changes because everyone touches everything all the time. The other usage site (`mesh_view_bindings.rs`) would require modifying more than one function, as you'd have to grab the OIT support state from the world and pass it around.